Stubbhult terrain.

Vlash271 opened the pull request:
## What type of PR is this.
1. [ ] Bug
2. [ ] Change
3. [x] Enhancement

### What have you changed and why?
Information: Adds the terrain [`Stubbhult`](https://steamcommunity.com/sharedfiles/filedetails/?id=3353189981) to the playable map pool. Versions created by Poki and SSGEnfisk were practically identical apart from vehicle markers positions, resources + factories and a couple seaports instead of outposts. Worked off Poki's version as airfields were already built up with some extra defences. I have mostly cleaned up some of the markers, added radio towers as the map had none inside any outposts and removed or moved certain towns.

General map information: 

- 12.3km x 12.3km
- 4 airbases
- 26 towns
- 20 outposts
- 4 seaports
- 29 resources
- 4 factories

### Please specify which Issue this PR Resolves.
closes #XXXX

### Please verify the following and ensure all checks are completed.

1. [x] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [ ] No
2. [x] Yes (Please provide further detail below.)

General playtesting with a higher player population.

### How can the changes be tested?
Steps: 

********************************************************
Notes:

Vlash271 committed:
Base Files for Antistasi Stubbhult map

Vlash271 committed:
Merge branch 'unstable' of https://github.com/Vlash271/A3-Antistasi into Stubbhult

Vlash271 committed:
mapInfo edits, general mission edits.

Vlash271 committed:
Town blacklist, police stations, town config changes

Vlash271 committed:
Missing bracket

Vlash271 committed:
Merge branch 'official-antistasi-community:unstable' into Stubbhult

Vlash271 committed:
HQ moved, added Map menu UI info

Vlash271 committed:
I can't count - town number changed

Vlash271 committed:
Plane marker clean up.

Vlash271 committed:
navGrid clean up

Vlash271 commented:
![20251116132120_1](https://github.com/user-attachments/assets/636eb2f0-41d6-4485-9b0c-70ac1f3f7eab)

Vlash271 committed:
Merge branch 'official-antistasi-community:unstable' into Stubbhult

jaj22 commented:
Will be reviewing this shortly.

jaj22 commented:
Ok, some general issues:
- Player units should be switched to a3a_unit_player now, and A3A_core dependency added.
- Airbase spawn points seem to be placed fairly randomly. Spawn points should generally be placed near the road exit of the airbase. They exist because for airbases it's often difficult to determine the correct exit navpoint automatically due to distances & multiple potential exits.
- 3.10 automatically generates runway spawn places so adding places manually on runways may collide. The code does not check for overlaps.

Airbases:
airport_1:
- Six vehicle places total is a bit thin.

airport_3:
- Could reduce the size and place an airport_3_flag marker manually now.

airport_4:
- Probably too large. If you're 1km from the marker then you may be <200m from marker units & statics. Could split the helipad end into a separate outpost. Not ideal but preferable. Vehicle places are also a bit far.

jaj22 commented:
Outposts:
- Lack of HMG spots in general. Rather than sticking the small cargo towers everywhere, it might be preferable to add HMG places to some of those Livonia buildings. I'm thinking the multi-story guardhouse at least. But some 
- Vehicle places are often thin (too many 5x5s that only fit one vehicle) or have complex exit routes. I need to decide whether it's the first or last vehicle place that gets used for QRF & attack vehicle spawns.
- This map has a lot of outposts with complex internal road networks. Probably the best bet with these is to strip out most of the internal roads from the navgrid. I'm assuming that this wasn't done already. I haven't checked the navgrid yet.

outpost_1:
- Potential exit pathfinding issue.

outpost_2:
- Potential exit pathfinding issue.

outpost_4:
- Two vehicle places total.

outpost_6:
- Could really do with an MG tower to make it look more outposty.

outpost_8:
- One vehicle place.

outpost_9:
- Two vehicle places.

outpost_10:
- One vehicle place.

outpost_19:
- Exit road runs pretty close to hut.

outpost_24:
- Second vehicle marker probably can't drive past the first.

outpost_26:
- One vehicle place.

jaj22 committed:
Improve stubbhult navgrid

jaj22 committed:
Merge branch 'unstable' into Stubbhult

jaj22 committed:
Stubbhult updates:
- Split up long airfield.
- Remove outpost internal roads from navgrid.
- Clean up navgrid junctions a bit.
- Convert to A3A player units & correct vehicle box.
- Add HMG buildings to some outposts.
- Add a lot of camp positions.
- Improve some town positions.

Tiny-DM reviewed (approved):
rubber stamp
